src/Components/App/App.js

```diff
@@ -79,6 +77,12 @@ const App = () => {
             />
             }}
           />
+          <Route render={ () => {
+            return <ErrorPage 
+              message={'serverError'}
```

Comment:
Do you mean to pass a string to this errorPage? Is serverError supposed to be a variable?

Reply:
Oops, this is a placeholder for when I was going to render the error page on its own. I now have it rendering inside of the search component, so I can take this route out of switch all together.